internal/applications/notification/providers.go

```
@@ -125,12 +113,15 @@ func ProvideNotificationController(i do.Injector) (*notifController.Notification
 func ProvideBlastProducer(i do.Injector) (rabbitmq.MessageProducer, error) {
 	conn, err := do.Invoke[*rabbitmq.Connection](i)
 	if err != nil || conn == nil {
-		return nil, nil // Queue disabled — producer is nil; NotificationService.Blast returns an error when called
+		return nil, nil
```

golangci-lint (2.11.4)

[error] error is not nil (line 114) but it returns nil

(nilerr)